WPMedia/fusion-news-theme-blocks
# THEMES-1086: Move resizer URL

## Description

This PR updates the resizer URL so it comes from site properties instead of an environment variable. This is to prevent customers needing to change where this is set when upgrading to 2.0.

## Jira Ticket

- [THEMES-1086](https://arcpublishing.atlassian.net/browse/THEMES-1086)

## Acceptance Criteria

The resizer url is a site property in Themes Settings, not an env variable

## Test Steps

- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout THEMES-1086`
2. Checkout the same branch for `arc-themes-components` and `arc-themes-feature-pack` 
3. Run fusion repo with linked blocks `npx fusion start -f -l`
4. On the all blocks test page, verify that images are still loading, and using the resizer URL.

## Dependencies or Side Effects

- Themes Components PR: https://github.com/WPMedia/arc-themes-components/pull/179
  - After this PR is merged, this branch will need to be updated with new `arc-themes-components` version in the `package-lock.json` for Chromatic tests to pass.
- Feature Pack PR: https://github.com/WPMedia/arc-themes-feature-pack/pull/401

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1086]: https://arcpublishing.atlassian.net/browse/THEMES-1086?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
